vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -331,31 +352,33 @@ type internal FSharpQuickInfoProvider
 
                     // get whitespace nomalized documentation text
                     let getText (tts: seq<Layout.TaggedText>) = 
-                        ((StringBuilder(), tts) ||> Seq.fold (fun sb tt -> 
-                            if String.IsNullOrWhiteSpace tt.Text then sb else sb.Append tt.Text)).ToString() 
+                        let text = 
+                            (StringBuilder(), tts) 
+                            ||> Seq.fold (fun sb tt -> 
+                                if String.IsNullOrWhiteSpace tt.Text then sb else sb.Append tt.Text)
+                            |> string
+                        if String.IsNullOrWhiteSpace text then None else Some text
                     
                     let documentation = 
-                        let implText, sigText = getText targetDocumentation, getText  sigDocumentation
-                        let implDocsEmpty, sigDocsEmpty = String.IsNullOrWhiteSpace implText, String.IsNullOrWhiteSpace sigText
-                        
-                        [ match implDocsEmpty, sigDocsEmpty with
-                          | true, true -> ()
-                          | true, false -> yield! sigDocumentation
-                          | false, true -> yield! targetDocumentation
-                          | false, false when implText.Equals (sigText, StringComparison.OrdinalIgnoreCase) -> yield! sigDocumentation
-                          | false, false -> 
+                        [ match getText targetDocumentation, getText sigDocumentation with
+                          | None, None -> ()
+                          | None, Some _ -> yield! sigDocumentation
+                          | Some _, None -> yield! targetDocumentation
+                          | Some implText, Some sigText when implText.Equals (sigText, StringComparison.OrdinalIgnoreCase) -> 
+                              yield! sigDocumentation
+                          | Some _, Some _ -> 
```

Comment:
I think this is cleaner
```fsharp
                        match implDocsEmpty, sigDocsEmpty with
                        | true, true -> Seq.empty
                        | true, false -> upcast sigDocumentation
                        | false, true -> upcast targetDocumentation
                        | false, false when implText.Equals (sigText, StringComparison.OrdinalIgnoreCase) -> upcast sigDocumentation
                        | false, false -> 
                          seq{    
                            yield! sigDocumentation
                            yield lineBreak
                            yield seperator
                            yield lineBreak
                            yield! targetDocumentation }
```
and converting the resizeArray to a list to cast it to a seq is unnecessary, just cast it straight to a seq